backend/code_groups/cg_controllers.js

```diff
@@ -76,7 +75,7 @@ exports.createGroup = async (req, res) => {
           create: members
             ? members.map((memberUsername) => ({
                 user: { connect: { username: memberUsername } },
-                addedBy: { connect: { id: creatorId } },
+                addedById: creatorId,
```

Comment:
curious about this change, why setting the id directly instead of setting the "connect" param?

Reply:
I removed the connection from the database and just added a addedById field that takes an Int value. I was thinking along the line of "less is more"; since I can always use addedById to get the user's details if I ever needed to.